docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -101,6 +101,91 @@ NOTE: The properties marked with *bold* are mandatory.
+As well as the suites, profiles are loaded one by one starting from the last one in the sequence. This allows to reuse common properties and separate specific ones.
+
+==== Profiles setup
+
+Define the number of profiles you need to set up. In case of multiple profiles, distinguish common and specific properties and put them into `profile.properties` files in appropriate places.
+
+Let's create an example. Imagine that our project contains automated scenarios for a couple of platforms: web and mobile. We will need to run autotests on different browsers and different devices. Therefore some of the properties would be common, but many of them should only belong to a certain subject. Let's see profiles structure for this case:
+
+[source,dir,subs=attributes+]
+----
+└─ src/main/resources/properties
+                            └─  profile <1>
+                                    ├─ web <2>
+                                        ├─ chrome <4>
+                                             └─ profile.properties <6>
+                                        ├─ firefox <4>
+                                        ├─ safari <4>
+                                        └─ profile.properties <5>
+                                    ├─ mobile_app <2>
+                                            ├─ iOS <7>
+                                            ├─ android <7>
+                                                └─ profile.properties <9>
+                                            └─ profile.properties <8>
+                                    └─ profile.properties <3>
+
+----
+<1> Profile directory which contains all the profiles properties;
+<2> General type of the profiles, either a browser or a mobile device;
+<3> The highest level of profile properties. Values written in this file will be applied to any browser or device;
+<4> Web browsers' profiles (directories);
+<5> Web-oriented properties. Values from this file will be applied for any web browser, but not for mobiles;
+<6> Chrome-specific profile. Any properties written here will not affect any other browsers or mobile devices;
+<7> Mobile profiles (directories);
+<8> Mobile-oriented properties. Values from this file will be applied for iOS and Android devices only;
+<9> Android-specific profile. Any properties written here will not affect iOS devices or any browser setup;
+
+By default, profile property is pointed to the `/src/main/resources/properties/profile` directory, so use the relative path starting from `profile` when you have nested directories:
+
+.configuration.properties
+[source,properties]
+----
+configuration.profiles=mobile_app/android
+----
+
+==== Build-in profiles
+
+Some of the plugins, like xref:plugins:plugin-web-app.adoc[web-app] have build-in profiles, you might find it useful to check it in the Referenced libraries:
+
+.Click to view plugin profiles
+[%collapsible]
+====
+image::web-app-profiles.png[]
+====
```

Comment:
```suggestion
```
let's remove it:
1) it's plugin specific and should be described in the corresponding plugin
2) images are hard to update, so even in plugin docs I would put just textual description of the available plugins and the way to find them (but it's out-of-scope of the current PR)

Reply:
I left in place the approximate path to the profile directory since the paragraph doesn't make much sense without any additional information.